addons/web/static/src/js/fields/field_utils.js

```diff
@@ -330,8 +332,8 @@ function formatX2Many(value) {
  *        digits precision in the field. Note: if the currency defines a
  *        precision, the currency's one is used.
  * @param {boolean} [options.forceString=false]
- *        if false, returns a string encoding the html formatted value (with
- *        whitespace encoded as '&nbsp;')
+ *        if true, returns a string with regular whitespace. Otherwise it uses
+ *        non-breaking whitespace utf character
```

Comment:
@odoo/rd-framework-js 
Do you agree with such changes in the function definition ?

Reply:
utf => unicode
I'd also extend the comment to explain that this is done for historical reasons, and that using NBSP all the time should be valid (but we don't want to change the API in stable)